.golangci.yml

```diff
@@ -156,14 +158,6 @@ run:
   # directory holds the correct copies of dependencies and ignores
   # the dependency descriptions in go.mod.
   #
-  # Allowed values: readonly|vendor|mod
-  # By default, it isn't set.
-  modules-download-mode: readonly
-  # Allow multiple parallel golangci-lint instances running.
-  # If false (default) - golangci-lint acquires file lock on start.
-  allow-parallel-runners: false
```

Comment:
Now that you have removed parallel runners from all places would it run serially or parallely ?

Reply:
IMO ! We need this https://github.com/golangci/golangci-lint/issues/869 ! This is the root cause failures after updating the golang image to 1.23.0